apps/web/src/modules/dao/components/MembersList/MemberListCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Flex, Text } from '@zoralabs/zord'
+import dayjs from 'dayjs'
+import Link from 'next/link'
+import React, { useMemo } from 'react'
+
+import { Avatar } from 'src/components/Avatar'
+import { useEnsData } from 'src/hooks'
+
+import { DaoMember } from './MembersList'
+import { firstRowItem, lastRowItem, row, rowItem } from './MembersList.css'
+
+export const MemberCard = ({
+  member,
+  totalSupply,
+  isMobile,
+}: {
+  member: DaoMember
+  totalSupply?: number
+  isMobile: boolean
+}) => {
+  const { displayName, ensAvatar } = useEnsData(member.id)
+
+  const timeJoined = useMemo(
+    () => dayjs(dayjs.unix(member.timeJoined)).format('MMM DD, YYYY'),
+    [member]
+  )
+
+  const votePercent = useMemo(() => {
+    if (!totalSupply || !member.daoTokenCount) return '--'
+    return ((Number(member.daoTokenCount) / totalSupply) * 100).toFixed(2)
+  }, [totalSupply, member])
+
+  return (
+    <>
+      {isMobile ? (
```

Comment:
nit: can we potentially turn this return into one block instead of two for mobile vs desktop. you can apply styles at different screen sizes for zord components like this: `<Flex mt={{ '@initial': 'x16', '@768': 'x24' }} />`

Reply:
I was able to use the Zora styling for part of the issue. Still had to use a ternary. 


```ts
export const MemberCard = ({
  member,
  totalSupply,
  isMobile,
}: {
  member: DaoMember
  totalSupply?: number
  isMobile: boolean
}) => {
  const { displayName, ensAvatar } = useEnsData(member.address)

  const timeJoined = useMemo(
    () => dayjs(dayjs.unix(member.timeJoined)).format('MMM DD, YYYY'),
    [member]
  )

  const votePercent = useMemo(() => {
    if (!totalSupply || !member.daoTokenCount) return '--'
    return ((Number(member.daoTokenCount) / totalSupply) * 100).toFixed(2)
  }, [totalSupply, member])

  return (
    <Link href={`/profile/${member.address}`} passHref>
      <Flex mb={'x14'} direction={{ '@initial': 'column', '@768': 'row' }}>
        <Flex className={firstRowItem} align={'center'} mb={'x4'}>
          <Avatar address={member.address} src={ensAvatar} size="32" />
          <Text mx="x2" variant="paragraph-md">
            {displayName}
          </Text>
        </Flex>
        {isMobile ? (
          <Flex>
            <Text className={rowItem}>{member.daoTokenCount} Tokens</Text>
            <Text className={rowItem}>{votePercent}%</Text>
            <Text className={lastRowItem}>{timeJoined}</Text>
          </Flex>
        ) : (
          <>
            <Text className={rowItem}>{member.daoTokenCount} Tokens</Text>
            <Text className={rowItem}>{votePercent}%</Text>
            <Text className={lastRowItem}>{timeJoined}</Text>
          </>
        )}
      </Flex>
    </Link>
  )
}

```

Reason is that the percentages calculated in a parent box couldn't align with the percentages in the ruler box, as they could always collapse in separate proportion if the width of the parent box is different than the ruler. 

Thinking this doesn't require a large rewrite of the styling and component structure, and it also is now only one block instead of two. 

If that doesn't work for you, I can try rewriting the list with a grid-based layout.